### PR TITLE
Add samples for keyvalue mapper

### PR DESCRIPTION
**Pull request by chamathabeysinghe**

### CODE REVIEW

File: modules/samples/artifacts/0049/README.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1. Copy {WSO2SPHome}/samples/artifacts/0008/keyvalue-mapper-sample.siddhi file to {WSO2SPHome}/deployment/siddhi-files
```
Comment (minudika) [inline modules/samples/artifacts/0049/README.txt]: **{WSO2_SP_Home}/deployment/siddhi-files/** should be updated as **{WSO2_SP_Home}/wso2/worker/deployment/siddhi-files/**

**Comment by niruhan** [comment]
you need to add the simulation config (.jason) and .csv files if u have used any

File: modules/samples/artifacts/0049/keyvalue-mapper-sample.siddhi [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+@source(type='tcp', context='SmartHomeData', port='9892',
+@map(type='binary'))
+define stream SmartHomeData (houseId int, maxVal float, minVal float, avgVal double);
+
+@sink(type='inMemory', topic='home', @map(type='keyvalue', 
+@payload(a="""houseId""",b="""maxVal""", c ="""minVal""", d="""avgVal""")))
+define stream InMemorySmartHomeInputData (houseId int, maxVal float, minVal float, avgVal double);
+
+@source(type='inMemory', topic='home', @map(type='keyvalue', @attributes(houseId ="a", maxVal = "b", minVal = "c", avgVal = "d")))
```
Comment (tishan89) [inline modules/samples/artifacts/0049/keyvalue-mapper-sample.siddhi]: Why do you need this in-memory source?

Comment (chamathabeysinghe) [reply modules/samples/artifacts/0049/keyvalue-mapper-sample.siddhi]: I add this to show the use of Source Mapper

Comment (tishan89) [reply modules/samples/artifacts/0049/keyvalue-mapper-sample.siddhi]: please fix as discussed

**Pull request merged**